code_of_conduct.md

```diff
@@ -2,7 +2,7 @@
 
 ## Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socioeconomic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
Could you add `socio-economic` to the allow-list? This code of conduct is a standard text coming from https://github.com/EthicalSource/contributor_covenant/tree/2.1, and would prefer not to modify it.

Reply:
Done.  This is a UK vs. US spelling issue.  Codespell leans towards US spellings.